src/Identity/IdentityServer/PersistedGrantStore.cs

```diff
@@ -47,28 +58,11 @@ public async Task RemoveAsync(string key)
 
     public async Task StoreAsync(PersistedGrant pGrant)
     {
-        var grant = ToGrant(pGrant);
+        var grant = _toGrant(pGrant);
         await _grantRepository.SaveAsync(grant);
```

Comment:
❓ No fallback storage? I suppose that's moot.

Reply:
I don't think so. The fallback in this case is to just handle the migration to a new system where we need to read from the fallback until not needed anymore.